content/cheatsheet.md

```diff
@@ -199,3 +201,17 @@ Warnings suggest that users think carefully before doing steps that can cause ir
 **Output**:
 
 {{% warning %}}Backup your data before erasing the hard disk!{{% /warning %}}
+
+## Embed Video
+
+To embed a video, use the `video` shortcode with the location of the video and a video title.
+
+```{{< video "/images/video/location.mp4" "Video title" >}}```
```

Comment:
@HarunD I'm trying to add this as a code snippet so others will know how to do it but Hugo renders it.

Any ideas?

Reply:
You have to add comments tag like this:
`{{</* video "/images/video/location.mp4" "Video title" */>}}`